### ISSUE TITLE
[NSData writeToURL:atomically:] converted incorrectly

### ISSUE THREAD

**Issue by cherryblossom000** [Swiftify-Corp/Swiftify#127]
```m
[NSData writeToURL:url atomically:YES]
```

currently gets converted to

```swift
Data.write(to: url, atomically: true)
```

which causes the compiler error

> Extra argument 'atomically' in call

[`Data.write(to:options:)`][1] could be used instead:

```swift
Data.write(to: url, options: .atomic)
```

However, that returns `Void` and throws when unsuccessful, whereas [`[NSData writeToURL:atomically]`][2] returns `NO` if unsuccessful. [`NSData.write(to:atomically)`][3] which returns a `Bool` could then be used instead of `Data`:

```swift
NSData.write(to: url, atomically: true)
```

[1]: https://developer.apple.com/documentation/foundation/data/1779858-write
[2]: https://developer.apple.com/documentation/foundation/nsdata/1415134-writetourl?language=objc
[3]: https://developer.apple.com/documentation/foundation/nsdata/1415134-write

**Comment by cherryblossom000** [comment]
@tomattoz @alex-swiftify Any updates on this?

**Comment by alex-swiftify** [comment]
@cherryblossom000 Thanks for the reminder! We'll prioritize this.

@tomattoz Please take a look at this at the earliest convenience.

**Comment by tomattoz** [comment]
@cherryblossom000 fixed this such way: NSData(data: data).write(to: url, atomically: true)

**Comment by tomattoz** [comment]
The fix will appear in 5.1.5 release.
